packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -60,11 +170,24 @@ export function OpLatencyView(): React.ReactElement {
 				}}
 				yAxisUnitDisplayName="ms"
 				// NOTE: Because Op Latency data is not yet available, this is a placeholder
-				dataSets={[]}
+				dataSets={[
+					durationOutboundBatchingData,
+					durationNetworkData,
+					durationInboundToProcessingData,
+				]}
 			/>
 			<div className={styles.graphAboutContainer}>
 				<div className={styles.flexColumn}>
 					<Subtitle1>About</Subtitle1>
+					{renderText && (
+						<Body1Strong>
+							{`Unsampled telemetry has not been enabled in Devtools. To do so, open the web console and set the Disable Sampling flag to true using the following command:
+					localStorage.setItem("Fluid.Telemetry.DisableSampling", "true");`}
+							&nbsp;
+							{`This flag is only intended for local development with Devtools and should not be enabled in production scenarios.`}
```

Comment:
Can you include a screenshot of how this looks? I think we might want some extra formatting (newlines, and ideally making the command look like a code block).
```suggestion
							{`Unsampled telemetry has not been enabled and is necessary for this feature to work as designed. To enable it, open the web console and set the Disable Sampling flag to true using the following command:
					localStorage.setItem("Fluid.Telemetry.DisableSampling", "true");`}
							&nbsp;
							{`IMPORTANT: This flag is only intended for local development with Devtools and should not be enabled in production scenarios.`}
```